modules/EvseV2G/sdp.cpp

```diff
@@ -138,9 +138,15 @@ int sdp_send_response(int sdp_socket, struct sdp_query* sdp_query) {
         return 1;
     }
 
+    using state_t = tls::Server::state_t;
+    const auto tls_server_state = sdp_query->v2g_ctx->tls_server->state();
+
+    const auto tls_server_available =
+        (tls_server_state == state_t::init_complete or tls_server_state == state_t::running) ? true : false;
```

Comment:
A ternary `? true : false` is basically a nop. 😉
```suggestion
        (tls_server_state == state_t::init_complete or tls_server_state == state_t::running);
```

Reply:
Yeah you are right :+1: changed that